prebuild.js

```diff
@@ -0,0 +1,30 @@
+const dayjs = require('dayjs');
+const fs = require('fs');
```

Comment:
As it comes from Node.js, you can prefix to let the other developers know that it is a module from Node.js and not `package.json`.

```suggestion
const fs = require('node:fs');
```

Reply:
Fixed! Did not knew about that feature :D